ui/abstract/src/ui-abstract/properties/PrimitiveTypes.ts

```diff
@@ -53,6 +53,11 @@ export namespace Primitives {
     parts: CompositePart[];
   }
 
-  // eslint-disable-next-line
-  export type Value = Text | String | ShortDate | Boolean | Numeric | Enum | Point | Composite;
+  /** Instance key type. */
+  export interface InstanceKey {
+    className: string;
+    id: Hexadecimal;
```

Comment:
I think it makes more sense to use `Id64String` type here

Reply:
Switched to `Id64String`.